aries-site/src/pages/components/avatar.mdx

```diff
@@ -0,0 +1,140 @@
+import { BestPracticeGroup, Example } from '../../layouts';
+import {
+  AvatarAccessibilityExample,
+  AvatarDarkModeExample,
+  AvatarDataTableExample,
+  AvatarDataTableIconExample,
+  AvatarPageHeaderExample,
+  AvatarSizePreviewExample,
+  MarketplaceNavigationalCards,
+} from '../../examples';
+
+## When to use
+
+Avatars are used to enable users to quickly identify an object, whether that be a specific user, account, or other entity.
+
+Avatars are always paired with additional information, such as a user's name, for better clarity. If possible, use an image or logo over initials.
+
+The consistent dimensions of an avatar ensure that readability and scanability of content surrounding the avatar is not compromised. For example, when scanning a [data table column that contains an avatar of a company’s logo](/avatar#datatable) and the company name, the company name stays aligned from one row to the next. 
```

Comment:
Should Avatar be capitalized here? I am seeing it both ways throughout the document. Not sure what the correct way is. It makes sense to me to capitalize it since it is the name of the component, but we should align to what we have been doing naming wise on other pages.
```suggestion
The consistent dimensions of an Avatar ensure that readability and scanability of content surrounding the avatar is not compromised. For example, when scanning a [data table column that contains an avatar of a company’s logo](/avatar#datatable) and the company name, the company name stays aligned from one row to the next. 
```

Reply:
I went through and changed avatar to not be capital to align with our guidance Daliha wrote